src/components/test/RegistrationModal.test.tsx

```diff
@@ -201,16 +200,6 @@ describe("RegistrationModal", () => {
 
         expect(registerSpy.mock.calls.length).toBe(1);
       });
-
-      it("select existing handle switches back to handle selection", () => {
-        component
-          .find({ children: "Select Existing Handle" })
-          .simulate("click");
-
-        expect(component.find("RegistrationModal").first().text()).toContain(
-          "Select an account:"
-        );
-      });
     });
   });
```

Comment:
write test to see that select registrations is there if there is more than 1 registration.